main.go

```diff
@@ -56,36 +56,44 @@ func main() {
 // If during execution, an unrecoverable error occurs (usually due to a bug),
 // an error message is logged and the program will panic.
 func Main(apps applications.AppMap, args []string, out io.Writer) int {
-	if len(args) < 2 {
+	if len(args) < 4 {
```

Comment:
I think if you'd write `--kind=Foobar` it's only 3, right?

Reply:
True, I changed it to 3, should be okay since there is a check later on if all key have their values and if --kind is present.